controllers/user.controller.ts

```diff
@@ -27,10 +27,40 @@ class UserController {
       email,
       password,
       role: role || roles[0],
+      isVerified: true,
```

Comment:
I think we need to think a bit here,
When an admin creates the user, isVerified property should be false? and then when user verifies his email we can set it to true? Let me know what you think. 
Also when admin is creating a user, password should be generated automatically with random words and send that password in email to user?

Reply:
I thought we should remove the parameter `password` and set `isVerified: false` for this case. Why administrator should set and know a password for a new user? 
What do you think?

Reply:
yes that's exactly what we need to do. So there are 2 user scenarios:
1. When user signs up by himself :  at the begining,  `isVerified` MUST be set to `false` and once he verifies email, `isVerified`  should be set to `true`
2. When Admin adds a user, at the begining,   `isVerified` MUST be set to `false` and password should be generated randomly and save as AES encrypted string in the db. and once User verifies the email, `isVerified`  should be set to `true`

Reply:
What about `isDisabled` in both cases?

Reply:
`isDisabled` is a property when we need to disable a user for some time. So it should be set to `false` initially